### PR TITLE
Fixed SMuFL anchors

### PR DESCRIPTION
**Pull request by cbjeukendrup**
Resolves: 
- https://github.com/musescore/MuseScore/issues/8952#issuecomment-907587882 (only @oktophonie's comment, not yet the whole issue)
- https://trello.com/c/86Ks7zZX/58-stemdownnw-stemupse-not-honoured-for-x-notehead

While working on #8952, I wrote some extra drawing code for debugging bboxes, positioning, anchors etc.. This revealed that the SMuFL anchors were hardly ever in the correct place, so stems were not attached correctly to noteheads. They always seemed to be at 80% of their correct position from the origin. And exactly that was the case. Now it looks like this: 
Schermafbeelding 2021-08-29 om 18 02 08
(red `+` means origin of element; orange `+` is flag anchor point; yellow `+` is notehead anchor point; the green rect is the bounding box. Please don't mind the badly-placed and purple-coloured stems in this picture.)

Here is it without those extra things:
Schermafbeelding 2021-08-29 om 18 38 26

It also turned out that the metadata file for Petaluma was not added to the application, so for Petaluma the anchors did not work at all. That's fixed too with this PR.

### CODE REVIEW

**Comment by Jojo-Schmitz** [comment]
What puzzels me is that there are no vtest differences repoted? There should be some minor one (but for the better)

**Comment by cbjeukendrup** [comment]
> What puzzles me is that there are no vtest differences reported? There should be some minor ones (but for the better)

That makes me a little worried too, also because I remember a PR of yours for which we also expected vtest differences but there wasn't. Makes me wonder whether the vtests are working at all...

**Comment by Jojo-Schmitz** [comment]
Yes, #8853

**Comment by cbjeukendrup** [comment]
It probably starts going wrong because the vtest input files are still old mscx files. It is unclear to me whether any output images are generated at all.

Then the compare step silently fails:
```
Run ./vtest/vtest.sh -c compare
MuseScore VTest
RUN_CMD: compare
SCORES_DIR: ./vtest/scores
ARTIFACTS_DIR: vtest.artifacts
MSCORE_BIN: build.debug/install/bin/mscore
DPI: 130
Compare PNG files and references
ls: cannot access 'vtest.artifacts/ref/*.ref.png': No such file or directory
```

Will try to fix it tomorrow.

**Comment by Jojo-Schmitz** [comment]
It is a separate issue though, not the business of this PR here. Which is good as it is, and should get merged sooner rather than later ;-)

**Review by oktophonie — APPROVED** [review]

File: src/importexport/musicxml/internal/musicxml/importxmlfirstpass.cpp [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 #include "importxmlfirstpass.h"
 
+#include "log.h"
```
Comment (vpereverzev) [inline src/importexport/musicxml/internal/musicxml/importxmlfirstpass.cpp:25]: is it really in use here?

**Review by vpereverzev — APPROVED** [review]

File: src/importexport/musicxml/internal/musicxml/importxmlfirstpass.cpp (hunk 1 above)
Comment (cbjeukendrup) [reply src/importexport/musicxml/internal/musicxml/importxmlfirstpass.cpp:25]: Yes, `IF_ASSERT_FAILED()`, in line 176.

**Pull request merged**